RELEASE/scripts/autoscend/paths/legacy_of_loathing.ash

```diff
@@ -145,8 +171,15 @@ boolean lol_buyReplicas()
 			}
 			else
 			{
-				buy($coinmaster[Replica Mr. Store], 1, $item[replica deactivated nanobots]);
-				use(1, $item[replica deactivated nanobots]); // put in terrarium
+				if(!is100FamRun()) //If this isn't a 100% familiar run, go ahead and get another familiar
```

Comment:
To come full circle, this logic is ok in my mind since the 100fam check is in the else of the check for resolutions. It means we already bought resolutions so only need to support 1 more pick